src/classes/vscommands/i18ncommand/ExportToI18NCommand.ts

```diff
@@ -220,7 +220,14 @@ export class ExportToI18NCommand extends ParserBearer {
 	private async _insertIntoI18NFile(stringToInsert: string) {
 		const [manifest] = this._parser.fileReader.getAllManifests();
 		const manifestFsPath = manifest?.fsPath;
-		const i18nRelativePath = manifest?.content["sap.app"]?.i18n;
+		let i18nRelativePath = manifest?.content["sap.app"]?.i18n;
+		if(typeof i18nRelativePath === "object"){
+			i18nRelativePath = i18nRelativePath.bundleUrl
+		}
+		if(!i18nRelativePath){
+			throw new Error("Inavlid i18n bundle path valid formats are a string or an object with a property 'bundleUrl'")

```

Comment:
Please change the message to:
"Inavlid i18n bundle path in manifest.json. Please define path to i18n by setting 'sap.app.i18n' or 'sap.app.i18n.bundleUrl' field if manifest.json"

Reply:
I will take your wording I wish we could just link the ui5 docu here but sadly the links to the docu are not stable